plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,795 @@
+      country:
+        title: Country
+        description: You can optionally provide us with the default country or countries(comma separated) this phone number is suspected to be associated with Our system will prefer to use a country on this list for verification or will require a country to be specified in the event the phone number is less than 10 digits
+        type: string
+        required: false
+        example: 'US'
+    output:
+      formatted:
+        title: Formatted
+        description: The phone number formatted in the international dialing code NA if not formattable
+        type: string
+        required: false
+        example: '0'
+      local_format:
+        title: Local Format
+        description: The phone number formatted in the country's local routing rules with area code NA if not formattable
+        type: string
+        required: false
+        example: '0'
+      valid:
+        title: Valid
+        description: Is the phone number properly formatted and considered valid based on assigned phone numbers available to carriers in that country?
+        type: boolean
+        required: false
+        example: false
+      fraud_score:
+        title: Fraud Score
+        description: The IPQS risk score which estimates how likely a phone number is to be fraudulent Scores 85+ are high risk
+        type: integer
+        required: false
+        example: 0
+      recent_abuse:
+        title: Recent Abuse
+        description: Has this phone number been associated with recent or ongoing fraud?
+        type: boolean
+        required: false
+        example: false
+      VOIP:
+        title: VOIP
+        description: Is this phone number a Voice Over Internet Protocol (VOIP) or digital phone number?
+        type: boolean
+        required: false
+        example: false
+      prepaid:
+        title: Prepaid
+        description: Is this phone number associated with a prepaid service plan?
+        type: boolean
+        required: false
+        example: false
+      risky:
+        title: Risky
+        description: Is this phone number associated with fraudulent activity, scams, robo calls, fake accounts, or other unfriendly behavior?
+        type: boolean
+        required: false
+        example: false
+      active:
+        title: Active
+        description: Is this phone number a live usable phone number that is currently active?
+        type: boolean
+        required: false
+        example: false
+      carrier:
+        title: Carrier
+        description: The carrier (service provider) this phone number has been assigned to or NA if unknown
+        type: string
+        required: false
+        example: 'airtel'
+      line_type:
+        title: Line Type
+        description: The type of line this phone number is associated with (Toll Free, Mobile, Landline, Satellite, VOIP, Premium Rate, Pager, etc) or NA if unknown
+        type: string
+        required: false
+        example: 'Unknown'
```

Comment:
Can this example please be updated to reflect the description value (NA instead of Unknown).